### PR TITLE
Use valid defaults for host and domain in default config file

### LINKED ISSUE #71: Using $host and $domain makes an invalid configfile when not replaced
($host and $domain can be used for Appconfig's variable expansion, but they are invalid ccm.conf parameters)

Part of https://github.com/quattor/release/issues/109#issuecomment-144091675

### PR DESCRIPTION
**Pull request by stdweird**
($host and $domain can be used for Appconfig's variable expansion, but they are invalid ccm.conf parameters)

Fixes #71

### CODE REVIEW

**Comment by stdweird** [comment]
hmm, so there is a unittest that checks the validity of the example configfile. need to reinvestigate the crash of https://github.com/quattor/release/issues/109#issuecomment-144091675

**Comment by stdweird** [comment]
The unittest failed because `ccm.conf` is both config file for `ccm-fetch` as an application (so parsed by CAF::Application / Appconfig) and for `CCM` parsed by CCfg (which supports its own version of variable expansion via _resolveTags). 
we should really stop writing our own configfileparsers ...

File: src/test/perl/test-ccfg.t [hunk 1]
```diff
@@ -28,6 +28,25 @@ my $ec = LC::Exception::Context->new->will_store_errors;
 
 # test the resolveTags  method
 is(EDG::WP4::CCM::CCfg::_resolveTags("a"),       "a",     "_resolveTags(a)");
+
+is(EDG::WP4::CCM::CCfg::_resolveTags('__HOST__'),   $host,   '_resolveTags(__HOST__)');
+is(EDG::WP4::CCM::CCfg::_resolveTags('__DOMAIN__'), $domain, '_resolveTags(__DOMAIN__)');
+is(EDG::WP4::CCM::CCfg::_resolveTags('__HOST____DOMAIN__'), $host . $domain,
+    '_resolveTags(__HOST____DOMAIN__)');
```
Comment (piojo) [inline src/test/perl/test-ccfg.t]: I can haz descriptive, story-telling names for the test cases?

Comment (piojo) [inline src/test/perl/test-ccfg.t]: For instance, 

```perl
is(EDG::WP4::CCM::CCfg::_resolveTags('__HOST__'),   $host, '__HOST__  tag correctly expanded'); 
```

**Pull request merged**